src/main/java/com/mappy/fpm/batches/tomtom/helpers/RoadTagger.java

```diff
@@ -74,25 +73,21 @@ public RoadTagger(SpeedProfiles speedProfiles, GeocodeProvider geocodeProvider,
 
         Long id = feature.getLong("ID");
 
+        Map<String, String> directionTags = restrictionTagger.tag(feature);
```

Comment:
Traffic flow direction logic was previously done in this file. We now delegate it to the restriction tagger, and transfer the oneway and reversed information to the methods needing it.